test/Microsoft.AspNet.Mvc.Razor.Host.Test/Directives/ChunkInheritanceUtilityTest.cs

```diff
@@ -36,14 +35,15 @@ @inherits MyBaseType
             var utility = new ChunkInheritanceUtility(host, cache, defaultChunks);
 
             // Act
-            var chunkTrees = utility.GetInheritedChunkTrees(PlatformNormalizer.NormalizePath(@"Views\home\Index.cshtml"));
+            var chunkTreeResults = utility.GetInheritedChunkTreeResults(
+                PlatformNormalizer.NormalizePath(@"Views\home\Index.cshtml"));
 
             // Assert
-            Assert.Collection(chunkTrees,
-                chunkTree =>
+            Assert.Collection(chunkTreeResults,
+                chunkTreeResult =>
                 {
                     var viewImportsPath = PlatformNormalizer.NormalizePath(@"Views\home\_ViewImports.cshtml");
-                    Assert.Collection(chunkTree.Chunks,
```

Comment:
Could you throw in an `Assert.Equal(viewImportsPath, chunkTreeResult.FilePath);` here?